source/conf.py

```diff
@@ -17,8 +17,10 @@
 # -- Project information -----------------------------------------------------
 from typing import List
 
-project = 'OpenQASM Live Specification'
-copyright = '2017-2020, Andrew W. Cross, Lev S. Bishop, John A. Smolin, Jay M. Gambetta'
+version = os.getenv('VERSION','Live')
+
+project = f'OpenQASM {version} Specification'
+copyright = '2017-2023, Andrew W. Cross, Lev S. Bishop, John A. Smolin, Jay M. Gambetta'
```

Comment:
Not a new problem introduced here, but at some point we need to investigate the proper way to claim copyright to the OpenQASM community, Qiskit, or whatever entity makes sense from a legal point of view.

Reply:
Added a new issue here: https://github.com/openqasm/openqasm/issues/466